examples/ethernet/ethernet/main/lan8720_phy.c

```diff
@@ -0,0 +1,125 @@
+
+#include "esp_attr.h"
+#include "esp_log.h"
+#include "esp_eth.h"
+
+#include "lan8720_phy.h"
+
+extern char *demo_tag;
```

Comment:
better to do something like `static const char *TAG = "lan8720";` here, similar for the other PHY.